pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -127,9 +127,25 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 		return api.Environment{}, err
 	}
 	setProductLabels(cr, &mergedEnv)
+	setCRBuildConfigEnvsToSpec(cr, &mergedEnv)
```

Comment:
any chance this can be moved into this method instead?
https://github.com/kiegroup/kie-cloud-operator/blob/d3cab494497a791b417306581623963cea7b2604/pkg/controller/kieapp/defaults/defaults.go#L618

Reply:
moved to this function.